todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/logic.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * -\-\-
+ * --
+ * Copyright (c) 2017-2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+@file:JvmName("StatisticsLogic")
```

Comment:
probably not necessary since it won't be used from Java after you convert the unit tests

Reply:
@anawara: When converting this to Kotlin, is the plan to only convert the existing tests to Kotlin? Would this be a good time for us to also go through and improve test quality/coverage as needed?